### PR TITLE
Fix warning and add 'swiftier' syntax in Demo project

### PR DESCRIPTION
**Pull request by albinekcom**
- Fix the warning "A 83.5x83.5@2x app icon is required for iPad apps targeting iOS 9.0 and later" in Demo project
- Add "swifter" syntax in Demo project

### CODE REVIEW

**Comment by khandpur** [comment]
Hi Albin, thanks for sending this PR our way. Please sign the CLA so we can start reviewing it.

**Comment by albinekcom** [comment]
I signed it!

File: Demo/EarlGreyExample/Images.xcassets/AppIcon.appiconset/Contents.json [hunk 1]
```diff
@@ -63,10 +63,16 @@
       "idiom" : "ipad",
       "filename" : "AppIcon-76-2x.png",
       "scale" : "2x"
+    },
+    {
+      "size" : "83.5x83.5",
```
Comment (tirodkar) [inline Demo/EarlGreyExample/Images.xcassets/AppIcon.appiconset/Contents.json]: How was this image asset generated?

Comment (albinekcom) [inline Demo/EarlGreyExample/Images.xcassets/AppIcon.appiconset/Contents.json]: It is scaled down the `AppIcon-60-3x.png` file.

Comment (khandpur) [inline Demo/EarlGreyExample/Images.xcassets/AppIcon.appiconset/Contents.json]: LGTM from my end.
@tirodkar please merge if this looks good to you.

**Pull request merged**